stock_picking_show_lot/views/stock_picking_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_stock_picking_show_lot" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button
+                    string="Show Lots"
+                    type="object"
+                    name="action_show_lots"
+                    class="oe_stat_button"
+                    icon="fa-cubes"
+                    attrs="{'invisible': [('use_create_lots', '=', False)]}"
```

Comment:
Revome, I think that we can show it for all pickings... 
Use show_lots_text field to determine the visibility..

Reply:
Perhaps it is better to define a boolean calculated field to control visibility based on whether or not there are lots in the picking. If we use the show_lots_text field we limit the visibility if it has been validated, plus the "use_existing_lots" option has to be disabled.

https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_picking.py#L537

Reply:
More easier!! display button if user has the security group...

Reply:
done!